HISTORY.md

```diff
@@ -1,4 +1,6 @@
-# Next
+# Release 0.25.0
```

Comment:
0.26.0

Reply:
Thanks, looks like my last push with this fix failed and I never made it back to that terminal. So will re-run the wheel build to be sure.